server/services/transactionsTransformer.go

```diff
@@ -221,8 +217,19 @@ func (transformer *transactionsTransformer) normalTxToRosetta(tx *transaction.Ap
 	}, nil
 }
 
-// This only handles operations for the native balance.
-func (transformer *transactionsTransformer) extractInnerTxOperationsIfRelayedCompletelyIntrashardWithSignalError(tx *transaction.ApiTransactionResult) ([]*types.Operation, error) {
+// extractInnerTxOperationsIfBeforeSiriusRelayedCompletelyIntrashardWithSignalError recovers the inner transaction operations (native balance transfers)
+// for a transaction that is relayed and completely intrashard, and has a signal error, and was processed before the activation of Sirius features.
+// Before Sirius, such a transaction is accompanied by an SCR with the value refund, thus Rosetta has to recover the inner transaction operations, as well,
+// to show the complete picture (that the operations cancel each other out).
+// After Sirius, the protocol does not generate an SCR with the value refund for such transactions, so this workaround is not needed.
+// Additional references:
+// - https://github.com/multiversx/mx-chain-rosetta/pull/81/files
+// - https://console.cloud.google.com/bigquery?sq=667383445384:bfeb7de9aeec453192612ddc7fa9d94e
+func (transformer *transactionsTransformer) extractInnerTxOperationsIfBeforeSiriusRelayedCompletelyIntrashardWithSignalError(tx *transaction.ApiTransactionResult) ([]*types.Operation, error) {
+	if transformer.provider.IsReleaseSiriusActive(tx.Epoch) {
```

Comment:
Ideally, we will remove `extractInnerTxOperationsIfBeforeSiriusRelayedCompletelyIntrashardWithSignalError` sometimes soon, because, most probably, reconciliation for such old transactions isn't necessary.